### ISSUE TITLE
Request for help: how to move Z-Probe to Analog input A4?

### ISSUE THREAD

**Issue by abrilevskiy** [gnea/grbl#850]
Hello,

I accidentally damaged A5 input and trying to move functionality of Z-Probe to A4 GPIO.
Could you please review code modification I made and let me know if that is all/OK?

cpu_map.h file is only modified in the following way:

`96c96
<   #define PROBE_BIT       5  // Uno Analog Pin 5
---
>   #define PROBE_BIT       4  // Uno Analog Pin 5
107c107
<     #define COOLANT_MIST_BIT   4  // Uno Analog Pin 4
---
>     #define COOLANT_MIST_BIT   5  // Uno Analog Pin 4
164c164
<       #define DUAL_STEP_BIT       4  // Uno Analog Pin 4
---
>       #define DUAL_STEP_BIT       5  // Uno Analog Pin 4
239c239
<       #define COOLANT_FLOOD_BIT   4  // Uno Analog Pin 4
---
>       #define COOLANT_FLOOD_BIT   5  // Uno Analog Pin 4
`

Thanks,
Andrei.

**Comment by doppelhub** [comment]
Yeah that's all you need to do to swap pins.  Note: This only works because both A5 & A4 are on the same port (PORTC).  If that weren't the case, then you'd need to change more things.

I suppose that isn't working, hence your post?
